README.md

```diff
@@ -79,4 +79,35 @@ The local PostgreSQL database is exposed with the port 5432. You can connect to
 user: admin
 password: admin_pw
 ```
-Make sure you don't have anything else running on this port. The credentials above are default and defined in `docker-compose.yml`
\ No newline at end of file
+Make sure you don't have anything else running on this port. The credentials above are default and defined in `docker-compose.yml`
+
+### Debug with Debugpy in Visual Studio Code
+
+1. Open the project in Visual Studio Code.
+2. Install **[Python Debugger extension](https://code.visualstudio.com/docs/python/debugging)** inside VS Code
+3. At the project root folder, create .vscode/launch.json with the following content:
+    ```json
+   {
```

Comment:
This launch.json file can just be part of the commit rather than instructions. It shouldn't change for each user.